manager/integration/tests/test_basic.py

```diff
@@ -600,6 +600,45 @@ def backup_failure_predicate(b):
     cleanup_volume(client, volume)
 
 
+@pytest.mark.skip(reason="TODO")
+def test_backup_block_deletion():  # NOQA
+    """
+    Test backup block deletion
+
+    Context:
+
+    We want to make sure that we only delete non referenced backup blocks,
+    we also don't want to delete blocks while there other backups in progress.
+    The reason for this is that we don't yet know which blocks are required by
+    the in progress backup, so blocks deletion could lead to a faulty backup.
+
+    Setup:
+
+    1. Create a volume and attach to the current node
+    2. Setup NFS backupstore since we can manipulate the content easily
+
+    Steps:
+
+    1.  Write 2 backup blocks of data on to the volume
+    2.  Create backup(1) of the volume
+    3.  Overwrite the first of the backup blocks of data on the volume
+    4.  Create backup(2) of the volume
+    5.  Overwrite the first of the backup blocks of data on the volume
+    6.  Create backup(3) of the volume
+    7.  Verify backup block count == 4
+    8.  Create an artificial in progress backup.cfg file
```

Comment:
How to create the artificial backup.cfg?

Reply:
One can place an in progress backup.cfg into the backups folder for that volume via the nfs mount. An example can be found in the engine integration test, except for the nfs part.
An example of the nfs operations can be found in [here](https://github.com/longhorn/longhorn-tests/blob/099d46e9182e081dd76bb45011966d49968d0989/manager/integration/tests/test_basic.py#L1153)

Reply:
Don't need details on how to modify NFS backupstore, but need to specify what to be put in the file (e.g. a blank file? a file with some content?).